frontend/src/css/Sidebar.styl

```diff
@@ -85,10 +85,18 @@ sidebarColorDarken = darken(sidebarColor, 15%)
         height 100%
         &:hover
           background-color sidebarColorDarken
+        &__notChildren
```

Comment:
```suggestion
        &__noChild
```

Reply:
`noChildren` (or `withoutChildren`)

(zero is plural in English)